packages/suite/src/constants/suite/experimental.ts

```diff
@@ -1,15 +1,23 @@
 import { TranslationKey } from '@suite-common/intl-types';
+import { isDesktop } from '@trezor/env-utils';
 
-export enum ExperimentalFeature {
-    PasswordManager = 'password-manager',
-}
+export type ExperimentalFeature = 'password-manager' | 'silent-update';
 
-type FeatureIntlMap = Partial<Record<ExperimentalFeature, TranslationKey>>;
-
-export const ExperimentalFeatureTitle: FeatureIntlMap = {
-    [ExperimentalFeature.PasswordManager]: 'TR_EXPERIMENTAL_PASSWORD_MANAGER',
+export type ExperimentalFeatureConfig = {
+    title: TranslationKey;
+    description: TranslationKey;
+    isEnabled: () => boolean;
```

Comment:
I agree that `title` and `description` probably should be there for every feature, but what about `isEnabled`? It seems unnecessary to be forced to write `() => true` everywhere. Unless most of the features will have a condition like this :thinking:

If not, one solution may be to switch it to `isDisabled` (or even better to `isHidden`, so it's not mistaken with actually enabling/disabling a feature) and make it optional?

Reply:
I like `isDisabled` approach